### DISCUSSION TITLE
Installation Problems

### DISCUSSION THREAD

**Discussion by lnrdwsh1ngt0n** [BradyAJohnston/MolecularNodes#627] [General]
Hello Brady,

i was hoping to get started with MolecularNodes for Blender for an upcoming project. It seems like a great tool from what ive seen in your video tutorials. Unfortunately though, ive been having some problems with getting the extension to function to its full capabilities. I was hoping you, or anyone else in the forum could help me out! Heres my problem (I'm working on windows 10):
- I installed the current version of Blender (4.2.2)
- Then i downloaded MolecularNodes 4.2.7 from Github.
- Under _Edit - Preferences - Get extensions_ i selected _Install from disk_ and selected the MolecularNodes.zip 
- The installation works, and i can enable the Add-on in the _Preferences_ menu
![image](https://github.com/user-attachments/assets/57bf28c8-86d6-4653-95cf-7f6b5d23cdba)

Here's where i encounter the issue: I cannnot isntall the _Atomium_ Package in the MolecularNodes Add-on dropdown, as it does not appear there (Neither does MDAnalysis)
![image](https://github.com/user-attachments/assets/d82825d8-d44c-46d2-b78a-c9adf63ca768)

Im assuming that this is the reason MolecularNodes does not create a property collection
![image](https://github.com/user-attachments/assets/df70f6c3-0abd-480a-91ab-087646c0347b)

which in turn means that the geometry nodes look quite different than the ones you show in the tutorial.
![image](https://github.com/user-attachments/assets/3e2b8e70-2322-4c75-a638-d17deaf541f6)

This is the second PC ive tried this on, after de-installation and re-installation of both blender and MolecularNodes was unsuccessful on the first device. 
I would really appreciate any help or advice on what im doing wrong, as this basically prevents me from using the extension for the purpose i had in mind. If theres any additional information that is required for solving the issue, im happy to provide it. 

Best regards,
Leon

**Comment by BradyAJohnston** [comment]
The YouTube tutorial is for quite an outdated version. You no longer need to install Atomium (or any additional dependencies). Molecular Nodes can be installed from within Blender itself (without having to download from GitHub). 

Please consult the [online documentation](https://bradyajohnston.github.io/MolecularNodes/installation.html) for the most up to date information.
